lib/presentation/molecules/study_type_selection_molecule.dart

```diff
@@ -49,26 +49,28 @@ class _StudyTypeSelectionMoleculeState
             margin: false,
           ),
           const SeparatorAtom(variant: SeparatorVariant.farApart),
-          ListTileAtom(
-            TipType.analytical.name,
-            leading: Radio<TipType>(
-              value: TipType.analytical,
-              groupValue: selectedType,
-              onChanged: onChanged,
-            ),
-            subtitle: 'recommended_morning',
-          ),
-          ListTileAtom(
-            TipType.creative.name,
-            leading: Radio<TipType>(
-              value: TipType.creative,
-              groupValue: selectedType,
-              onChanged: onChanged,
-            ),
-            subtitle: 'recommended_evening',
-          ),
+          studyTypeRadioButton(onChanged, selectedType, TipType.analytical),
+          studyTypeRadioButton(onChanged, selectedType, TipType.creative),
         ],
       ),
     );
   }
+
+  Widget studyTypeRadioButton(void Function(TipType?)? onChanged,
```

Comment:
I think null is not necessary, we always transfer function and type
`void Function(TipType?)? `

Reply:
This fix is included in #51